lnrpc/rpc.proto

```diff
@@ -684,9 +684,7 @@ message Utxo {
 	string script_pubkey = 4 [json_name = "script_pubkey"];
 
 	/// The outpoint in format txid:n
-	/// Note that this reuses the `ChannelPoint` message but
-	/// is not actually a channel related outpoint, of course
-	ChannelPoint outpoint = 5 [json_name = "outpoint"];
+	OutPoint outpoint = 5 [json_name = "outpoint"];
```

Comment:
Can we remove message ChannelPoint completely? It it the same message now, and channel point is an outpoint too.

Reply:
Does it also break if you leave the `ChannelPoint` message in to not get errors from people building against the compiled proto, but really use the `OutPoint` type in all the calls? Not sure if the actual type is also serialized. If not, it may just work if they are identical.

Reply:
Not sure what you mean, but projects importing the protos will have to do the name change (similar to what is done in `rpcserver` here). The serialized version will be the same as long as the field number (`5`) is the same IIRC.

Reply:
Yes, I meant just keeping the ChannelPoint message in so that dependents don't need to rename and just keep working because ChannelPoint and OutPoint are identical. But maybe not such a good idea.